src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -105,79 +93,76 @@ const Header = ({
-          <View style={styles.headerDomainDateContainer}>
-            <Text style={styles.headerDomainDate} numberOfLines={1}>
-              {calculateTime(time)}
+          <View style={styles.wrapperImage(showBackButton)}>
+            {image ? (
+              <Image source={{uri: image}} style={[styles.image, StyleSheet.absoluteFillObject]} />
+            ) : (
+              <MemoDomainProfilePicture width="47" height="47" />
+            )}
+          </View>
+          <Gap width={SIZES.base} />
+          <View style={styles.headerDomainInfoContainer}>
+            <Text style={styles.headerDomainName} numberOfLines={1}>
+              {name}
             </Text>
-            <View style={styles.point} />
-            <MemoPeopleFollow height={13} width={12} />
-            <Gap style={{ width: 3.33 }} />
-            <Text style={styles.headerFollowerText}>12k</Text>
-            <View style={styles.point} />
-            <FeedCredderRating
-              containerStyle={styles.credderRating}
-              scoreSize={normalizeFontSize(12)}
-              scoreStyle={{marginTop: normalizeFontSize(1.5)}}
-              score={item?.domain?.credderScore}
-              iconSize={16} />
+            <View style={styles.headerDomainDateContainer}>
+              <Text style={styles.headerDomainDate} numberOfLines={1}>
+                {calculateTime(time)}
+              </Text>
+              <View style={styles.point} />
+              <MemoPeopleFollow height={13} width={12} />
+              <Gap style={{width: 3.33}} />
+              <Text style={styles.headerFollowerText}>12k</Text>
+              <View style={styles.point} />
+              <FeedCredderRating
+                containerStyle={styles.credderRating}
+                scoreSize={normalizeFontSize(12)}
+                scoreStyle={{marginTop: normalizeFontSize(1.5)}}
+                score={item?.domain?.credderScore}
+                iconSize={16}
+              />
+            </View>
           </View>
+        </Pressable>
+        <View style={{justifyContent: 'center'}}>
+          <TouchableOpacity onPress={onFollowDomainPressed}>
+            {follow ? (
+              <View style={styles.wrapperTextUnFollow}>
+                <MemoUnfollowDomain />
+              </View>
+            ) : (
+              <View style={styles.wrapperText}>
+                <MemoFollowDomain />
+              </View>
+            )}
+          </TouchableOpacity>
         </View>
-      </Pressable>
-      <View style={{ justifyContent: 'center' }}>
-        <TouchableOpacity onPress={onFollowDomainPressed}>
-          {follow ? (
-            <View style={styles.wrapperTextUnFollow}>
-              <MemoUnfollowDomain />
-            </View>
-          ) : (
-            <View style={styles.wrapperText}>
-              <MemoFollowDomain />
-            </View>
-          )}
-        </TouchableOpacity>
       </View>
-    </View>
     </SafeAreaView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk are mostly related to the color theme refactoring. However, there's a potential issue with the `wrapperImage` style. The borderColor has been changed from `'rgba(0,0,0,0.5)'` to `COLORS.black50`. Please ensure that `COLORS.black50` is equivalent to `'rgba(0,0,0,0.5)'` to maintain the same visual appearance.



<!-- This is an auto-generated comment by OSS CodeRabbit -->